crates/task-impls/src/quorum_proposal.rs

```diff
@@ -535,8 +535,8 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> QuorumProposalTaskState<TYPE
                         }
 
                         let mut consensus = self.consensus.write().await;
-                        if let Err(e) = consensus.update_high_qc(qc.clone()) {
-                            tracing::trace!("{e:?}");
+                        if consensus.update_high_qc(qc.clone()).is_err() {
+                            tracing::trace!("High QC with an equal or higher view exists.");
```

Comment:
To avoid needing to write this error every time since it looks like it was copied in another spot, we can put this message into the `ensure` statement. Same as the above custom trace messages, then we can just trace print the error. I think we can actually do `trace!(?e)` to avoid string templating.

Reply:
Yeah that's better. Done!